configure.ac

```diff
@@ -224,9 +224,9 @@ AC_ARG_ENABLE([zmq],
   [use_zmq=yes])
 AC_ARG_ENABLE([bip70],
   [AS_HELP_STRING([--disable-bip70],
```

Comment:
Should probably change the help string to --enable-bip70 so it is more comprehensible.